tests/foreman/cli/test_myaccount.py

```diff
@@ -72,61 +90,62 @@ def test_positive_update_language(self):
         @Status: Manual
         """
 
-    @stubbed()
     def test_positive_update_password(self):
-        """@Test: Update Password/Verify fields in My Account
+        """Update Password/Verify fields in My Account
```

Comment:
There is no verify in CLI right?  So this may just be """Update Password in My Account"""